_events/2019-11-14-scale-bythebay.md

```diff
@@ -2,7 +2,7 @@
 category: event
 title: Scale by the Bay
 logo: /resources/img/scalebythebay.png
-location: San Francisco, CA, USA
+location: Oakland, CA, USA
 description: ""
 start: 14 November 2019
 end: 16 November 2019
```

Comment:
Looks like the dates should be changed to 13-15 November as well

Reply:
good catch! I pushed a second commit with a fix